### PR TITLE
Release version 1.14 (#265)

### PR DESCRIPTION
**Pull request by whimboo**
This PR fixes issue #265. @davehunt would you mind to test the final bits of this version if it all works for you? I would like to have your feedback for I'm going to release it. Please also let me know if those changes are good. Thanks.

### CODE REVIEW

File: History.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+1.14 / 2015-03-05
+=================
+
+ * Allow download of files with specified extension for Tinderbox builds on Windows (#264)
+ * Replace --directoy option with --destination to allow a filename for the target file (#92)
```
Comment (davehunt) [inline History.md]: Typo: --directoy should be --directory

File: History.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
...
+ * Always show correct build number for candidate builds (#232)
+ * Add test for invalid branches of daily builds (#236)
+ * `mac` platform option for tinderbox builds should default to `macosx64` (#215)
+ * Reverse check for non-empty tinderbox build dirs (#253)
```
Comment (davehunt) [inline History.md]: Nit: I would personally avoid abbreviating 'directories'

**Comment by davehunt** [comment]
Other than the typo/nit, this looks good to me.

File: History.md (hunk 1 above)
Comment (whimboo) [inline History.md]: That's just the output from `git changelog --list` so the typing failure happened to me when I commited the changes. I can fix it for the release notes.

File: History.md (hunk 2 above)
Comment (whimboo) [inline History.md]: Same here as above. I will correct it in the release notes.

**Comment by whimboo** [comment]
Commits have been squashed and PR merged as https://github.com/mozilla/mozdownload/commit/028ae444426b6e7691138e88ac306c6f8e6dfd74.

**Pull request closed without merging**